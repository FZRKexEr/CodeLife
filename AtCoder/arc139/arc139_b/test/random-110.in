100
94 1 9 18 18 3
34 7 7 20 8 12
49 17 1 4 12 15
37 12 1 15 2 4
1 15 18 5 17 19
28 10 5 13 13 13
78 6 7 20 13 16
61 16 20 18 12 9
31 13 8 2 7 1
43 3 16 4 3 19
92 2 15 5 7 13
72 18 18 5 18 10
1 2 10 11 16 9
50 5 5 12 17 12
21 9 6 9 15 17
1 5 1 18 8 6
83 11 7 8 8 16
48 8 2 3 20 6
16 5 15 1 8 12
45 6 5 20 20 6
5 11 18 15 1 19
93 8 13 14 14 7
30 18 2 10 2 5
64 6 4 15 8 1
4 7 17 4 7 14
39 10 19 19 17 12
24 5 5 5 16 16
87 15 15 8 9 17
7 18 14 2 10 2
10 20 19 2 1 3
15 13 9 14 7 11
29 8 1 3 7 10
61 17 2 1 2 13
17 5 13 3 6 9
21 9 13 16 14 3
57 17 5 13 2 12
10 17 19 7 14 19
10 17 7 18 3 4
1 16 13 16 11 16
32 19 9 19 10 8
53 14 20 20 8 2
63 8 20 14 17 12
48 14 18 17 9 9
27 3 17 7 18 17
87 6 8 9 15 2
13 16 16 5 12 1
36 16 3 5 3 8
88 15 1 20 12 1
7 14 20 2 11 8
99 8 19 14 10 10
69 2 18 18 19 9
7 18 3 2 12 6
61 13 7 13 8 6
69 8 17 6 9 1
6 13 18 9 7 14
71 7 14 10 19 17
27 3 13 19 7 3
3 13 7 18 7 13
19 12 15 7 10 9
61 5 9 19 9 4
71 16 8 13 3 8
30 11 2 18 5 20
98 14 12 19 12 5
36 14 11 18 2 18
41 3 10 8 17 18
74 4 9 10 4 17
49 9 17 13 20 5
18 4 17 2 3 18
16 7 12 13 17 9
100 2 7 8 15 11
19 11 7 14 1 6
6 11 3 6 20 14
68 3 8 12 18 2
87 3 2 13 10 12
88 2 16 19 2 9
61 17 14 1 17 9
89 20 9 17 12 20
4 9 19 7 17 4
47 18 7 16 13 9
41 3 19 15 15 2
90 19 6 15 14 4
34 19 1 15 9 4
84 8 5 10 18 1
28 11 16 14 14 2
16 12 13 16 5 1
77 14 15 15 19 5
70 4 3 2 4 19
19 4 8 1 8 10
89 7 20 15 20 11
36 12 7 14 11 16
74 13 20 10 2 2
20 5 10 1 9 11
4 1 5 16 3 9
84 12 3 8 15 16
14 20 19 18 20 2
3 16 2 11 4 19
51 1 16 18 1 13
58 1 19 3 7 6
15 7 17 7 8 17
49 4 17 12 5 15
