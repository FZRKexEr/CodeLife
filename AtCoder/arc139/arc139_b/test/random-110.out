102
152
84
10
5
78
169
45
13
46
85
40
11
120
49
6
104
120
13
54
75
77
32
16
16
43
84
141
14
20
35
36
16
21
76
31
70
58
16
94
118
78
61
78
44
65
19
73
14
106
178
14
46
10
54
95
25
54
37
61
103
90
47
38
80
90
108
16
60
169
21
28
46
290
53
41
68
28
65
19
60
69
43
30
49
55
72
19
94
33
18
20
12
105
252
30
42
21
23
55
